### PR TITLE
Readme.rst updates

### LINKED ISSUE #1616: Fix README.md
Currently README.md is totally out of date, but many people encounter it.  Let's fix it!

### PR DESCRIPTION
**Pull request by pde**
Fixes #1616

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -17,38 +14,87 @@ https://letsencrypt.org. Be sure to checkout the
...
+<https://github.com/ietf-wg-acme/acme/blob/master/draft-ietf-acme-acme.md>`_
+protocol) that can automate the tasks of obtaining certificates and
+configuring webservers to use them.
+
+Installation
+------------
+
+If ``letsencrypt`` is packaged for your OS, you can install it from there, and
+run it by typing ``letsencrypt``.  Because not all operating systems have
+packages yet, we provide a temporary solution via the ``letsencrypt-auto``
```
Comment (bmw) [inline README.rst]: Should we wait to talk about `letsencrypt-auto` until @erikrose makes his changes? Starting tomorrow, these instructions are out of date.

File: README.rst [hunk 2]
```diff
@@ -17,38 +14,87 @@ https://letsencrypt.org. Be sure to checkout the
...
+wrapper script, which obtains some dependencies from your OS and puts others
+in an python virtual environment::
+
+  user@webserver:~$ git clone https://github.com/letsencrypt/letsencrypt
+  user@webserver:~$ cd letsencrypt
+  user@webserver:~/letsencrypt$ ./letsencrypt-auto --help
+
+Or for full command line help, type::
+
+  ./letsencrypt-auto --help all | less
```
Comment (bmw) [inline README.rst]: I personally don't like `| less` here, but I don't care too strongly.

File: README.rst [hunk 3]
```diff
@@ -3,12 +3,9 @@
...
 ==========
 
-This is a **DEVELOPER PREVIEW** intended for developers and testers only.
-
-**DO NOT RUN THIS CODE ON A PRODUCTION SERVER. IT WILL INSTALL CERTIFICATES
-SIGNED BY A TEST CA, AND WILL CAUSE CERT WARNINGS FOR USERS.**
-
-Browser-trusted certificates will be available in the coming months.
+The Let's Encrypt Client is **BETA SOFTWARE**. It contains plenty of bugs and
+rough edges, and should be tested thoroughly in staging evironments before use
```
Comment (bmw) [inline README.rst]: s/evironments/environments

**Comment by bmw** [comment]
Other than comments above, LGTM.

File: README.rst (hunk 1 above)
Comment (pde) [inline README.rst]: I'm fine with another micro-PR once `letsencrypt-auto` can be obtained by other means.

**Pull request merged**